src/main/resources/com/microfocus/application/automation/tools/run/RunFromFileBuilder/config.jelly

```diff
@@ -251,15 +251,28 @@
 				 <f:entry>
 					 <f:checkbox title="Use SSL" name="runfromfs.useSSL" checked="${instance.runFromFileModel.useSSL}" default="true"/>
 				 </f:entry>
-				 <f:entry title="User name" field="fsUserName">
-					 <f:textbox name="runfromfs.fsUserName" value="${instance.runFromFileModel.fsUserName}"/>
-				 </f:entry>
-				 <f:entry title="Password" field="fsPassword">
-					 <f:password name="runfromfs.fsPassword" value="${instance.runFromFileModel.fsPassword}"/>
-				 </f:entry>
-				 <f:entry title="Tenant Id" field="mcTenantId">
-					 <f:textbox name="runfromfs.mcTenantId" value="${instance.runFromFileModel.mcTenantId}"/>
-				 </f:entry>
+                <f:entry>
+                    <f:radioBlock name="authModel" field="authModel" value="token" checked="${instance.runFromFileModel.authType == 'token' || instance.runFromFileModel.authType == null}" title="Execution Token Authentication">
```

Comment:
Would introduce a new separate jelly model for this, as an example SpecifyParametersModel in the new PR, or UFTSettingsModel previously. The jelly modifications are duplicated, a new model would resolve this. If you plan to integrate this, please also introduce a separate JS file for the model, extracting all the code from the configure.js, regarding MC.